bootstrap.php

```diff
@@ -223,6 +241,62 @@
 $GLOBALS['processedRecords'] = 0;
 $GLOBALS['allProcessedRecords'] = 0;
 
+/*
+ * if user wants all modules, append system modules to existing config
+ */
+if (isset($opts['allmodules'])) {
+    global $moduleList;
+    foreach ($moduleList as $aModule) {
+        if (!isset($modules[$aModule])) {
+            $modules[$aModule] = (is_numeric($opts['a']) ? $opts['a'] : 5000);
+        }
+    }
+    ksort($modules);
+}
+
+/*
+ * if user wants all relationships, append to existing config
+ */
+if (isset($opts['allrelationships'])) {
+    $result = $GLOBALS['db']->query(' SELECT * FROM relationships WHERE deleted=0 ');
```

Comment:
Could you please replace it with ```global $dictionary;``` as you did in your initial commit (https://github.com/sugarcrm/Tidbit/pull/48/commits/ec9e6677a65d130688c18762b249aa6bf518cec9)? Seems like this table is not used any more in 7.x versions, but global sugar ```$dictionary``` contains all relationships information that we need.